.ado/windows-vs-pr.yml

```diff
@@ -676,14 +674,6 @@ jobs:
         inputs:
           script: yarn api
 
-      - script: |
```

Comment:
Just curious if the version were out of whack, why don't we make sure the version is 'fixed' and determined in the `Setup` step?

Reply:
I don't think we can super cleanly fix the version for JS publish, since Verdaccio would proxy to a registry that would already have the version published. You could maybe unpublish for Verdaccio only, and specialize Beachball to use a different branch than we were using for changefiles, but it gets kind of messy.